docs/UserGuide.adoc

```diff
@@ -47,214 +47,354 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
-[NOTE]
-====
-Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
-====
+****
+* The search is case insensitive. e.g `tom` will match `Tom`
+* Multiple names must be seperated by a comma. e.g. `/order find john, hilary`.
+* The order of the keywords does not matter. e.g. `Alice Smith` will match `Smith Alice`
+* Partial match is allowed e.g. `tom` will match `tommy`
+****
 
-// tag::undoredo[]
-=== Undoing previous command : `undo`
+Examples:
 
-Restores the address book to the state before the previous _undoable_ command was executed. +
-Format: `undo`
+* `/deliveryman find john` +
+Returns `john` and `John Doe`
+* `/order find john, alice, tom` +
+Returns `john`, `alice`, `tom`
 
-[NOTE]
-====
-Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit` and `clear`).
-====
+==== Deleting a delivery man : `/deliveryman delete`
+
+Deletes the specified delivery man from the list of delivery men +
+Format: `/deliveryman delete INDEX`
+
+****
+* Deletes a delivery at the specified `INDEX`.
+* The index refers to the index number shown in the displayed delivery men list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
 
 Examples:
 
-* `delete 1` +
-`list` +
-`undo` (reverses the `delete 1` command) +
+* `/deliveryman` +
+`/deliveryman delete 2` +
+Deletes the 2nd delivery man in the list of delivery men.
+* `/deliveryman find tom` +
+`delete 1` +
+Deletes the 1st delivery man in the results of the `find` command.
+
+==== Clearing all delivery men entries : `/deliveryman clear`
+
+Clears all entries from the list of delivery men. +
+Format: `/deliveryman clear`
+
+==== Creating a route : `/route create`
+
+Creates a new route and add it to the list of routes +
+Format: `/route create`
+
+==== Listing all delivery men : `/route`
+
+Shows a list of all existing routes and assigned delivery men +
+Format: `/route`
+
+==== Deleting a route: `/route delete`
+
+Deletes the specified route from the list of routes +
+Format: `/route delete INDEX`
 
-* `select 1` +
-`list` +
-`undo` +
-The `undo` command fails as there are no undoable commands executed previously.
+****
+* Deletes a route at the specified `INDEX`.
+* The index refers to the index number shown by `/route`.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
 
-* `delete 1` +
-`clear` +
-`undo` (reverses the `clear` command) +
-`undo` (reverses the `delete 1` command) +
+* `/route` +
+`/route delete 2` +
+Deletes the 2nd route in the list of routes.
 
-=== Redoing the previously undone command : `redo`
+==== Clearing all routes : `/route clear`
 
-Reverses the most recent `undo` command. +
-Format: `redo`
+Clears all entries from the list of routes. +
+Format: `/route clear`
+
+==== Adding an order to a route : `/plan`
+
+Add an existing order to an existing route +
+Format: `/plan` o/ORDER_INDEX r/ROUTE_INDEX
```

Comment:
I am not sure if this is how plan should work. I was thinking more like:
1. Create a route from orders
2. Add route to available deliveryman

What is plan doing?